piccolo/apps/user/commands/list.py

```diff
@@ -0,0 +1,26 @@
+from piccolo.apps.user.tables import BaseUser
+
+
+def list():
+    """
+    List existing users.
+    """
+    users = BaseUser.select(
+        BaseUser.all_columns(exclude=[BaseUser.password])
+    ).run_sync()
+
+    if len(users) < 1:
+        return
```

Comment:
Is it worth printing out something like 'No users'?

Reply:
Depends on how much of a purist you are. Personally, I'd just leave it with no output at all, but your call.

Reply:
My only concern with not printing something out is the user might think it's broken.

Reply:
Alright, I added a message.